cloudigrade/util/aws.py

```diff
@@ -370,6 +372,33 @@ def create_volume(snapshot_id, zone):
     return volume.id
 
 
+def get_volume(volume_id, region):
+    """
+    Return a Volume for an EC2 instance.
+
+    Args:
+        volume_id (str): A volume ID
+        region (str): The AWS region the volume exists in
+
+    Returns:
+        Volume: A boto3 EC2 Volume object.
+
+    """
+    return boto3.resource('ec2', region_name=region).Volume(volume_id)
+
+
+def check_volume_state(volume):
+    """Raise an error if volume is not available."""
+    if volume.state == 'creating':
+        raise AwsVolumeNotReadyError
```

Comment:
In case we have to investigate logs, let's put the volume id into the exception like you did with AwsVolumeError below.